analyzers/src/SonarAnalyzer.CSharp/Rules/InsecureEncryptionAlgorithm.cs

```diff
@@ -35,9 +35,11 @@ public sealed class InsecureEncryptionAlgorithm : DoNotCallInsecureSecurityAlgor
         // S2278 was deprecated in favor of S5547. Technically, there is no difference in the C# analyzer between
         // the 2 rules, but to be coherent with all the other languages, we still replace it with the new one
         private const string S2278DiagnosticId = "S2278";
+
         private const string S2278MessageFormat = "Use the recommended AES (Advanced Encryption Standard) instead.";
 
         private const string S5547DiagnosticId = "S5547";
+
         private const string S5547MessageFormat = "Use a strong cipher algorithm.";
 
         private static readonly DiagnosticDescriptor S2278 =
```

Comment:
This could be single line. Same for S5547. And then, the empty line between them will not be needed, as it's a regular and boring structure.